<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GiveBook</class>
 <widget class="QDialog" name="GiveBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>402</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="books" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>711</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="lineEdit_8">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Пошук</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_2">
        <item>
         <property name="text">
          <string>Назва</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Автор</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Рік</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Наявність</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Жанр</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Видавництво</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView_2"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Видати книгу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="readers" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>701</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Пошук</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_3">
        <item>
         <property name="text">
          <string>Ім'я</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Прізвище</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Номер телефону</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>День народження</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Вибрати читача</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="returnBook" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTableView" name="tableView_3"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Повернути книгу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="back2">
      <property name="text">
       <string>Назад</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>pushButton_6</zorder>
   <zorder>back2</zorder>
   <zorder>tableView_3</zorder>
  </widget>
  <widget class="QWidget" name="choose" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>691</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Видати книгу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Повернути книгу</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_5">
      <property name="text">
       <string>Завершити</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>returnBook</zorder>
  <zorder>readers</zorder>
  <zorder>choose</zorder>
  <zorder>books</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
